CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+## 1.38.1 (2020-02-14)
```

Comment:
Sorry for the slow review; it's already 15

Reply:
No worries.